docs/2_2_common_schema.adoc

```diff
@@ -1912,22 +1912,22 @@ In other words, the respective partial derivatives cannot be computed.
 |
 [[dependenciesKind, `dependenciesKind`]]
 If <<dependenciesKind>> is present, <<dependencies>> must be present and must have the same number of list elements.
-If not present, it must be assumed that the unknown latexmath:[\color{blue}{v_{\text{unknown}}}] depends on the knowns latexmath:[\color{blue}{\mathbf{v}_{\text{known}}}] without a particular structure.
-Otherwise, the corresponding known latexmath:[\color{blue}{v_{known,i}}] enters the equation as:
+If not present, it must be assumed that the unknown latexmath:[\color{blue}{v_{\mathit{unknow}}}}] depends on the knowns latexmath:[\color{blue}{\mathbf{v}_{\mathit{known}}}] without a particular structure.
+Otherwise, the corresponding known latexmath:[\color{blue}{v_{\mathit{known,i}}}] enters the equation as:
 
-`= dependent`: no particular structure, latexmath:[\color{blue}{{h(..,\ v}_{known,i}}],..)
+`= dependent`: no particular structure, latexmath:[\color{blue}{{h(..,\ v}_{\mathit{known,i}}}],..)
```

Comment:
Maybe this was find and replace magic, but 
`_{\mathit{known,i}}`
can be made more correct if transformed into:
`_{\mathit{known}, i}`
as `i` is a different element.

Reply:
fixed